[module:uninstall] Can't uninstall module if it is in the profile list

Hi, 

 I have a custom profile in my Drupal installation. This profile contains list of modules (https://github.com/DmitryDrozdik/dmaps/blob/master/d8/profiles/pp/pp.info.yml), which should be installed during installation Drupal profile. 

After successful installation, I can not uninstall module(dmaps), added to the profile list. But I should be able to do this. Uninstalling module works well on the Drupal "uninstall page".

This still seems to be an issue. I wanted to uninstall the core search module but was unable to do so with Drupal Console.

    $ drupal mou search
    
    [ERROR] Unable to uninstall modules "search" because are required by "standard"

Uninstalling via the admin/modules/uninstall page works fine.

--- a/src/Command/Module/UninstallCommand.php
+++ b/src/Command/Module/UninstallCommand.php
@@ -172,6 +172,7 @@ class UninstallCommand extends Command
         }
 
         if (!$force = $input->getOption('force')) {
+            $profile = drupal_get_profile();
             $dependencies = [];
             while (list($module) = each($moduleList)) {
                 foreach (array_keys($moduleData[$module]->required_by) as $dependency) {

